oauth2client/service_account.py

```diff
@@ -461,3 +465,94 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
+   epoch = datetime.datetime(1970, 1, 1)
+   time_delta = utc_time - epoch
+   return time_delta.days * 86400 + time_delta.seconds
+
+
+class JWTAccessCredentials(GoogleCredentials):
```

Comment:
I'm with @dhermes, I'd prefer this not be private. If I understand correctly, grpc will likely want to check if they have an instance of this class?